src/api/auth/use-forgot-password.ts

```diff
@@ -10,12 +10,16 @@ type Response = {
   message: string;
 };
 
+// Should be replaced with the app's web url.
+const redirectUrl = 'https://example.com';
+
```

Comment:
That could potentially be a deeplink but we can leave that for another feature ;)

Reply:
100%